homeassistant/components/swiss_public_transport/strings.json

```diff
@@ -17,11 +17,14 @@
         "data": {
           "from": "Start station",
           "to": "End station",
-          "via": "List of up to 5 via stations",
+          "via": "Via stations",
           "time_station": "Select the relevant station",
           "time_mode": "Select a time mode"
         },
         "data_description": {
+          "from": "The departure station for the start of the connection.",
+          "to": "The arrival station for the end of the connection.",
+          "via": "List of up to 5 via stations",
```

Comment:
I think we can make this nicer,

List of up to 5 stations the route must go through 

Like, one can only assume what a via station means

Reply:
@joostlek Yes clearer, I am not native english and already way too much biased as I am working on this integration AND IRL with public transport, it is hard to know the "basic" knowledge of a user we have to assume :)